src/api/.rubocop.yml

```diff
@@ -287,6 +287,11 @@ Rails/Output:
 # Checks for the use of methods which skip validations.
 Rails/SkipsModelValidations:
   Enabled: true
+  Exclude:
+    - 'db/migrate/*'
+    - 'app/jobs/*'
+    - 'spec/models/bs_request_action_spec.rb'
+    - 'spec/models/review_spec.rb'
```

Comment:
The last two, I wrote it in the description of the PR 

Also, there is an inconsistency in Rubocop (see https://github.com/bbatsov/rubocop/issues/4098) as it does not complain about the relation with `touch`, but it does complain about the test testing this. As I was not sure of what to do in this case I also added as an exception, but we could also change the relation.